capabilities.json

```diff
@@ -39,6 +39,12 @@
       "description": "Each facet instance is segmented by this field."
     },
     {
+      "name": "sparklineData",
+      "kind": 0,
+      "displayName": "Sparkline Axis",
+      "description": "A field to place on the horizontal axis for supporting spark line for the facet instance."
```

Comment:
comment nitpick: maybe "A field to render on the horizontal axis as a sparkline" ?